index.html

```diff
@@ -1,67 +1,99 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>판다마켓</title>
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
-    <header>
-        <div class="header">
-            <a href="/" >
-                <img src="images/ic_pandamarket_logo.png" alt="판다마켓 로고" class="logo"></a>
-            <a href="/login">
-                <img src="images/ic_login.png" alt="로그인" class="login"></a>
-        </div>
-    </header>
+<html lang="ko">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>판다마켓</title>
+        <link rel="stylesheet" href="style.css" />
+    </head>
+    <body>
+        <header>
+            <div class="header">
+                <a href="/"> <img src="images/logo.png" alt="판다마켓 로고" class="logo" /></a>
+                <a href="/login.html"> <img src="images/ic_login.png" alt="로그인" class="login" /></a>
```

Comment:
❗️ 수정요청
이미지보다 HTML + CSS로 구현하시는 것을 추천드려요~
이미지는 기본적으로 HTML보다 파일크기가 크고 반응형 처리가 어렵습니다!

```suggestion
                <a href="/login.html" class="login-button">로그인</a> // class를 추가해서 디자인을 구현하세요
```